code/modules/hydroponics/grown/misc.dm

```diff
@@ -137,3 +137,34 @@
 	playsound(src, 'sound/goonstation/misc/fuse.ogg', seed.potency, 0)
 	reagents.chem_temp = 1000 //Sets off the black powder
 	reagents.handle_reactions()
+
+	//cucumber
+	/obj/item/seeds/cucumber
+		name = "pack of cucumber seeds"
+		desc = "contains cucumber seeds"
+		icon_state = "seed-cucumber"
+		species = "potato" // have no proper grow/dead sprites for cucumber, decided to go with the potato ones for now. The rest works though.
+		plantname = "cucumber plant"
+		product = /obj/item/weapon/reagent_containers/food/snacks/grown/cucumber
+		lifespan = 20
+		endurance = 20
+		maturation = 10
+		production = 1
+		yield = 2
+		potency = 10
+		growthstages = 2
+		growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+		reagents_add = list("water" = 2)
```

Comment:
This will result in cucumbers being utterly worthless to eat on their own, as they provide only water. Plants don't automatically gain plantmatter or other forms of nutrients, you have to specify those here.

Was this intentional? It sure feels inconsistent.

Additionally, this is going to result in some SERIOUS amounts of water (could actually exceed the amount a grown object can hold before even reaching max potency). The number is a multiplier for the potency, so a potency 1 cucumber would be 2 units of water in this case. For comparison, watermelons are a 0.2 for water, meaning 0.2u per 1 potency.